### PR TITLE
fix: recursive cmd add reverse option

### PR DESCRIPTION
**Pull request by fireairforce**
closes: https://github.com/pnpm/pnpm/issues/3972

### CODE REVIEW

File: packages/config/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "test-with-preview": "ts-node test",
     "_test": "jest",
     "test": "pnpm run compile && pnpm run _test",
+    "start": "pnpm tsc --watch",
```
Comment (fireairforce) [inline packages/config/package.json:17]: The reason why i add this script is just for debug at local enviroment....

**Comment by zkochan** [comment]
I don't see changes in `exec.ts`. The #3972 is about adding reverse option to the exec command. Also, this option should not be added to `@pnpm/config`, just add it to the result of `rcOptionsTypes` in `run.ts` and `exec.ts`.

**Comment by fireairforce** [comment]
This changes happen in the commandName which is run.

**Comment by zkochan** [comment]
yes, you did it right in `run.ts`. You need to do the same in `exec.ts`

**Comment by fireairforce** [comment]
👌，i will fix it

**Comment by fireairforce** [comment]
Change as expected.

**Comment by zkochan** [comment]
But reverse will currently do nothing in `exec.ts`. It is not implemented there.

**Comment by fireairforce** [comment]
@zkochan Updated, PTAL if you are free.

**Review by zkochan — APPROVED** [review]

**Pull request merged**